tests/utils/test_performance_optimizer.py

```
@@ -1,176 +1,36 @@
-    result = await parser.process(['test', 'data'])
-    assert result == ['TEST', 'DATA']
-    
-    # Test cleanup
-    parser.cleanup()  # Should not raise any errors
\ No newline at end of file
+import unittest
+from unittest.mock import Mock, patch
+from src.utils.performance_optimizer import PerformanceOptimizer
+
+class TestPerformanceOptimizer(unittest.TestCase):
+    def setUp(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)